src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -628,7 +628,8 @@ void DoorLockServer::setCredentialCommandHandler(
 
         status = createCredential(commandPath.mEndpointId, fabricIdx, sourceNodeId, credentialIndex, credentialType,
                                   existingCredential, credentialData, userIndex, userStatus, userType, createdUserIndex);
-        sendSetCredentialResponse(commandObj, commandPath, status, createdUserIndex, nextAvailableCredentialSlot);
+
+        sendSetCredentialResponse(commandObj, commandPath, status, userIndex.Value(), nextAvailableCredentialSlot);
```

Comment:
This should be sending `createdUserIndex` like it used to, no?

Reply:
You're right, it does make mores sense to use the out param here.